dozer-ingestion/src/connectors/grpc/adapter/default.rs

```diff
@@ -142,8 +142,10 @@ fn map_record(rec: grpc_types::types::Record, schema: &Schema) -> Result<Record,
             ) => Ok(dozer_types::types::Field::Text(a.clone())),
             (
                 grpc_types::types::value::Value::BytesValue(a),
-                dozer_types::types::FieldType::Bson,
-            ) => Ok(dozer_types::types::Field::Bson(a.clone())),
+                dozer_types::types::FieldType::Json,
```

Comment:
I think we should expect an `google.protobuf.Value` here and convert `google.protobuf.Value` to `JsonValue`, just like we did in typed grpc service, but the other way around

Reply:
got it!